arm/compute/models.go

```diff
@@ -397,7 +374,6 @@ type AvailabilitySet struct {
 	Location                   *string             `json:"location,omitempty"`
 	Tags                       *map[string]*string `json:"tags,omitempty"`
 	*AvailabilitySetProperties `json:"properties,omitempty"`
-	Sku                        *Sku `json:"sku,omitempty"`
```

Comment:
Sorry - missed this! Digging our use - we're setting the SKU to `aligned` which [according to the initial PR](https://github.com/hashicorp/terraform/pull/12532) means the Availability Set can contain VM's with Managed Disks?